examples/js/loaders/MTLLoader.js

```diff
@@ -464,7 +464,10 @@ THREE.MTLLoader.MaterialCreator.prototype = {
 
 					if ( n > 0 ) {
 
-						params.opacity = 1 - n;
+						if ( this.options && this.options.equalizeTransparency )
```

Comment:
Just a code style issue. Please write it like this
```javascript
if ( this.options && this.options.equalizeTransparency ) {

   params.opacity = n;

} else {

   params.opacity = 1 - n;

}

Reply:
ok, I will update soon